SQF/dayz_code/stringtable.xml

```diff
@@ -13697,9 +13788,9 @@
 			<Russian>Сейф</Russian>
 		</Key>
 		<Key ID="STR_EPOCH_SAFE_DESC">
-			<English>A placeable object that is used for very secure storage. Placing your safe will give you a 4 digit number that is used to open the safe, save that number!</English>
-			<German>Ein platzierbares Objekt für sehr sichere Aufbewahrung. Der vierstellige Code zum Öffnen wird nach dem Aufstellen angezeigt, bitte nicht vergessen!</German>
-			<Russian>Личный сейф: Перемещаемый объект, который используется в качестве безопасного хранилища. ВНИМАНИЕ!!! Поставив сейф, вы получите 4-х значный код, который необходимо ввести, чтобы открыть сейф, запишите этот код!</Russian>
+			<English>A placeable object that is used for very secure storage. Placing your safe will give you a 4 digit number that is used to open the safe.</English>
+			<German>Ein platzierbares Objekt für sehr sichere Aufbewahrung. Der vierstellige Code zum Öffnen wird nach dem Aufstellen angezeigt.</German>
```

Comment:
Please remove the "sehr", it sounds stupid.
"für sichere Aufbewahrung" or "für eine sichere Aufbewahrung"